modules/imaging_browser/help/viewSession.md

```diff
@@ -1,6 +1,6 @@
 # Imaging Browser: View Session
 
-This page displays a candidate's scans collected from a single timepoint. Click on any image to launch the BrainBrowser visualization tool. With every file displayed, there are different actions you can take, such as change the file's QC status, add or edit the QC comments, download the MINC file, or see a longitudinal view (in BrainBrowser). Click **Header Info** to see further scan parameters from the DICOM header.
+This page displays a candidate's scans collected from a single timepoint. Click on any image to launch the BrainBrowser visualization tool. With every file displayed, there are different actions you can take, such as change the file's QC status, add or edit the QC comments, download the imaging files (MINC/NIfTI), or see a longitudinal view (in BrainBrowser). Click **Header Info** to see further scan parameters from the DICOM header.
```

Comment:
I find the "with every file displayed" wording confusing but it was there before so it's beyond the scope of this PR..